e2e/provision/README.md

```diff
@@ -106,6 +106,11 @@ connection with that setting).
        template:
          annotations:
            approval.nephio.org/policy: initial
+         pipeline:
```

Comment:
Question: Are L89-L113 captured as a KRM resource in the https://github.com/nephio-project/nephio-example-packages repo or any other repo?
Would it make sense?

Reply:
We could, yeah. Then the workflow would be to manually clone a pkg containing this resource into the mgmt repo, edit the resource for the clusters you want, and publish it.

We could in fact include this and resources in steps 9 and 11 and even more all in one package, and you could deploy everything: clusters, operator, free5gc-cp, and smf/upf/amf all in a single action of publishing one package.